.github/workflows/publish.yml

```diff
@@ -33,7 +33,7 @@ jobs:
     - name: Set up Go
       uses: actions/setup-go@v2
       with:
-        go-version: 1.18
+        go-version: 1.19.0
```

Comment:
similar to above, we can use 1.19 and the setup-go action will install the latest patch version